--- evdev.lst.base
+++ evdev.lst.patched
@@ -185,20 +185,22 @@
   unitekkb1925    Unitek KB-1925
   compalfl90      FL90
   creativedw7000  Creative Desktop Wireless 7000
   teck227         Truly Ergonomic 227
   teck229         Truly Ergonomic 229
   apex300         SteelSeries Apex 300 (Apex RAW)
   chromebook      Chromebook
 
 ! layout
   us              English (US)
+  us_ext          English (US + DE, IS, PL, Nordic)
+  ru_us           Russian (phonetic, US-based)
   af              Afghani
   ara             Arabic
   al              Albanian
   am              Armenian
   at              German (Austria)
   au              English (Australian)
   az              Azerbaijani
   by              Belarusian
   be              Belgian
   bd              Bangla
