--- evdev.xml.base
+++ evdev.xml.patched
@@ -2969,20 +2969,48 @@
             <description>Polish (British keyboard)</description>
             <languageList>
               <iso639Id>pol</iso639Id>
             </languageList>
           </configItem>
         </variant>
       </variantList>
     </layout>
     <layout>
       <configItem>
+        <name>ru_us</name>
+        <!-- Keyboard indicator for Russian layouts -->
+        <shortDescription>rux</shortDescription>
+        <description>Russian (phonetic, US-based)</description>
+        <countryList>
+          <iso3166Id>RU</iso3166Id>
+        </countryList>
+        <languageList>
+          <iso639Id>rus</iso639Id>
+        </languageList>
+      </configItem>
+    </layout>
+    <layout>
+      <configItem>
+        <name>us_ext</name>
+        <!-- Keyboard indicator for English layouts -->
+        <shortDescription>enx</shortDescription>
+        <description>English (US + DE, IS, PL, Nordic)</description>
+        <countryList>
+          <iso3166Id>US</iso3166Id>
+        </countryList>
+        <languageList>
+          <iso639Id>eng</iso639Id>
+        </languageList>
+      </configItem>
+    </layout>
+    <layout>
+      <configItem>
         <name>us</name>
         <!-- Keyboard indicator for English layouts -->
         <shortDescription>en</shortDescription>
         <description>English (US)</description>
         <countryList>
           <iso3166Id>US</iso3166Id>
         </countryList>
         <languageList>
           <iso639Id>eng</iso639Id>
         </languageList>
