--- evdev.xml.base
+++ evdev.xml.patched
@@ -3019,6 +3019,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>us_ext</name>
+            <description>English (US + DE, IS, PL, Nordic)</description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>mac</name>
             <description>English (Macintosh)</description>
           </configItem>
